builders/manage/sudo/impersonate.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Impersonating Other Accounts With Sudo 
+description: Learn how to use the Sudo key to dispatch calls by signing a transaction as if it came from a different account.
+---
+
+# Using Sudo to Impersonate Other Accounts
+
+## Introduction {: #introduction }
+
+[Sudo](https://paritytech.github.io/polkadot-sdk/master/pallet_sudo/index.html){target=\_blank} is a module that enables privileged runtime calls to be dispatched when called from the Sudo account. Sudo is sometimes colloquially referred to as a superuser or god-like account. This allows you to perform privileged actions while managing your appchain, such as impersonating other accounts. 
+
+In this guide, you'll learn how to use Sudo to impersonate other accounts. As an example, this guide will use the Sudo account to pose as an arbitrary account and transfer funds from said account.
+
+!!! warning
+    The balance transfer demonstrated in this guide is dubious, shown only as an example of using Sudo. 
+
+## Checking Prerequisites {: #checking-prerequisites }
+
+For the examples in this guide, you will need to have the following:
+
+ - A Tanssi appchain (Snap or Dedicated)
+ - Your appchain's Sudo account connected to your appchain's Polkadot.js Apps. You can refer to the [Managing Sudo guide](/builders/manage/sudo/sudo/#configuring-polkadotjs-apps){target=\_blank} for instructions on injecting your Sudo account into Polkadot.js Apps
+
+If you're unsure what your Tanssi appchain's Sudo account is, you can find it in your [Tanssi Dashboard](https://apps.tanssi.network/){target=\_blank} underneath the **Properties** section.
+
+![Locating your Sudo address on apps.tanssi.network](/images/builders/manage/sudo/impersonate/impersonate-1.webp)
+
+!!! warning
+    You should always protect your Sudo account key with the utmost security precautions, as it grants privileged access to your Tanssi appchain.
+
+## Using Sudo As {: #using-sudo-as }
```

Comment:
I don't use the Sudo pallet, and so the first time I read this, I thought the title got chopped off or something but then I kept reading and realized quickly that wasn't the case. However, I think the name should be changed to maybe "Using the Sudo As Method" or something